src/Documentation/glossary.js

```diff
@@ -21,10 +21,20 @@ form part of your expanded workspace, technically.
       name: 'DVC Project',
       match: ['DVC project', 'project', 'projects'],
       desc: `
-Initialized by running \`dvc init\` in the **workspace**. It will contain the
+Initialized by running \`dvc init\` in the **workspace** (typically in a Git
+repository). It will contain the
 [\`.dvc/\` directory](/doc/user-guide/dvc-files-and-directories) and
 [DVC-files](/doc/user-guide/dvc-file-format) created with commands such as
-\`dvc add\` or \`dvc run\`. It's typically also a Git repository.
+\`dvc add\` or \`dvc run\`. It may also be a Git repository.
+      `
+    },
+    {
+      name: 'DVC Repository',
+      match: ['DVC repository', 'DVC repositories'],
+      desc: `
+**DVC project** initialized using \`dvc init\` in a Git repository. It will
```

Comment:
it's _usually_ Git repo ... but DVC repo can be stand-alone

Reply:
If it's standalone you would call it **DVC project**. If it's a Git repo you can still call it project, but also **DVC repo**. Makes sense? We typically use the term _project_ except when we want to emphasize that its also a Git repo, especially when hosted (on GitHub).

Reply:
not sure I follow the logic tbh :)

Reply:
DVC repo is a kind of DVC project, when `git init && dvc init`. We usually don't need this term since we can just say DVC project, but some times it's important to emphasize that the project is also a Git repo.

The term "repository" was already used in the docs with this meaning, I'm' just tying give it a definition and use it consistently as "DVC repo".

Reply:
at the same time there's one doc where "DVC repository" refers to iterative/dvc, so another option is for me to completely remove this special term and any special use of "repository" in this context (replacing with something like "DVC project in a Git repo" or "Git + DVC project".

Reply:
It's still pretty confusing. To me DVC project is more or less the same as DVC repo. And may or may not use Git. In most cases it runs on top of Git though.

Reply:
Extracted to #782